### PR TITLE
kubescape 2.0.147 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by carlocab — DISMISSED** [review]

File: Formula/kubescape.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      -X github.com/armosec/kubescape/cautils.BuildNumber=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags)
+
+    output = Utils.safe_popen_read(bin/"kubescape", "completion", "bash")
+    (bash_completion/"kubescape").write output
+    output = Utils.safe_popen_read(bin/"kubescape", "completion", "zsh")
+    (zsh_completion/"_kubescape").write output
+    output = Utils.safe_popen_read(bin/"kubescape", "completion", "fish")
+    (fish_completion/"kubescape").write output
```
Comment (branchvincent) [inline Formula/kubescape.rb:23]:
```suggestion
    (fish_completion/"kubescape.fish").write output
```

File: Formula/kubescape.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  end
+
+  test do
+    manifest = "https://raw.githubusercontent.com/GoogleCloudPlatform/microservices-demo/main/release/kubernetes-manifests.yaml"
```
Comment (branchvincent) [inline Formula/kubescape.rb:27]:
```suggestion
    manifest = "https://raw.githubusercontent.com/GoogleCloudPlatform/microservices-demo/b8fe8900ca1da10c85c9a203d9832b2ee33cc85f/release/kubernetes-manifests.yaml"
```

File: Formula/kubescape.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/kubescape.rb]: Nice catch! Thanks!

**Review by branchv — APPROVED** [review]
👍

**Pull request closed without merging**